pages/report.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Weekly report"
+date: "December 04, 2024"
```

Comment:
```suggestion
```

Remove this since a different file updates and we should reference the date there.